Tests/HaCWebsiteLibTests/HaCWebsiteLibTestCase.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import XCTest
+
+class HaCWebsiteLibTestCase: XCTestCase {
```

Comment:
I agree it's non-ideal but I'm struggling to find a tidier option. The reason in this case is for the use of self.recordFailure, a method on XCTestCase which is the thing that actually triggers the failure. Granted, the test would fail if we just had throwing functions, but it's not as clear in the report what actually went wrong. Alternatives welcomed